bridge-status-bot/app.sh

```diff
@@ -43,6 +43,7 @@ GOLD_BALANCE=$(get_gold_balance "$BRIDGE_NC_ADDRESS")
 ETH_BALANCE=$(get_eth_balance "$BRIDGE_ETHEREUM_ADDRESS")
 ETH_BALANCE_THRESHOLD=3
```

Comment:
1> We might change `ETH_BALANCE_THRESHOLD` to 2 ETH from 3ETH. 
2> Also I think we could change `ETH_SAFE_EXECUTOR_THRESHOLD to 2`

Please discuss with Swen to change those value.